src/base/profile.cpp

```diff
@@ -43,7 +43,6 @@ Profile::Profile(Private::Profile *impl, Private::PathConverter *pathConverter)
     ensureDirectoryExists(SpecialFolder::Cache);
     ensureDirectoryExists(SpecialFolder::Config);
     ensureDirectoryExists(SpecialFolder::Data);
-    ensureDirectoryExists(SpecialFolder::Downloads);
```

Comment:
@Chocobo1:  Newbie question on `ensureDirectoryExists`:  
Prior to this change, what happens if the `Downloads` folder is both missing and not able to be created (e.g. if there is a blank file with the same name and location as the folder is supposed to have)?  

Would that prevent qBittorrent from starting?

Reply:
@Chocobo1:  

> Would that prevent qBittorrent from starting?
 
Just tested.  Apparently, the answer is "yes".  

Now that I have figured that out, I can finally use qBittorrent 3.4+.

Reply:
> Apparently, the answer is "yes".

Yep!

Reply:
>>Would that prevent qBittorrent from starting?
>
>Just tested. Apparently, the answer is "yes".
>
>Now that I have figured that out, I can finally use qBittorrent 3.4+!

I didn't understand, do you like current behavior (qBittorrent ensures that Download directory exists) or the fixed one (qBittorrent doesn't check for Download directory)?

Reply:
@glassez:  

> do you like current behavior (qBittorrent ensures that Download directory exists) or the fixed one (qBittorrent doesn't check for Download directory)?  

What I like, is that this PR informs me of **why** I could not get any version of qBittorrent after `3.4.0` to launch on my Windows machine.  

Since I store all of my personal files on a separate 'data' partition, I had blocked the creation/usage of all OS-defined folders (other than `AppData`) in my main user directory—including `Documents` and `Downloads`—by replacing them all with empty files with the same names.  

I did not consider that a versatile program such as qBittorrent might turn one of those folders into a dependency, and thus had no clue why qBittorrent was silently failing to launch (no matter what method I used to open it).